neo4j-to-neptune/pom.xml

```diff
@@ -47,6 +47,35 @@
             <version>2.10.5</version>
         </dependency>
 
+        <!-- https://mvnrepository.com/artifact/org.neo4j.driver/neo4j-java-driver -->
```

Comment:
Note: We should ensure we are compliant with AWS open source distribution requirements when this gets released. We likely need some sort of attribution notice, but I know they've been changing that policy recently so I don't recall the specifics